spec/requests/create_collection_spec.rb

```diff
@@ -212,10 +220,32 @@
 
           it 'saves the draft collection' do
             post '/collections', params: { collection: draft_collection_params, commit: save_draft_button }
+            expect(response).to have_http_status(:found)
             collection = Collection.last
             expect(collection.name).to be_empty
             expect(collection.depositors.size).to eq 0
+            expect(response).to redirect_to(collection_path(collection))
+          end
+        end
+
+        context 'with an invalid contact email' do
+          let(:draft_collection_params) do
+            {
+              name: '',
+              description: '',
+              contact_emails_attributes: invalid_contact_email,
```

Comment:
Since this let is only used in this place, can you move it into this context?

Reply:
sure, actually just ditched the let since the value was small and fits inline